src/Umbraco.AuthorizedServices/ClientApp/src/backoffice/AuthorizedServices/authorizedservice.constants.ts

```diff
@@ -0,0 +1,6 @@
+export enum AuthenticationMethod {
+  OAuth1 = "OAuth1",
+  OAuth2 = "OAuth2",
```

Comment:
Wondering if this should have a more detailed name  `OAuth2AuthorizationCode` perhaps?

Reply:
Updated